handsontable/hyperformula
# New license preamble

### Context


- replace full GPL license text in `LICENSE.txt` with the preamble
- include the preamble in all build artifacts (`hyperformula.js`, `hyperformula.full.js`, `hyperformula.min.js` etc)
- replace the term `commercial license` with `proprietary license` everywhere in the docs`

Internal discussion about it: https://handsoncode.slack.com/archives/C031GBX62SE/p1720691793462529

### How did you test your changes?


- build package locally and check the content of the build artifacts
- run docs locally and verify the updated links

### Types of changes

- [ ] Breaking change (a fix or a feature because of which an existing functionality doesn't work as expected anymore)
- [ ] New feature or improvement (a non-breaking change that adds functionality)
- [ ] Additional language file, or a change to an existing language file (translations)
- [x] Change to the documentation

### Checklist:


- [x] I have reviewed the guidelines about [Contributing to HyperFormula](https://hyperformula.handsontable.com/guide/contributing.html) and I confirm that my code follows the code style of this project.
- [ ] I have signed the [Contributor License Agreement](https://goo.gl/forms/yuutGuN0RjsikVpM2).
- [x] My change is compliant with the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html) standard.
- [x] My change is compatible with Microsoft Excel.
- [x] My change is compatible with Google Sheets.
- [ ] I described my changes in the [CHANGELOG.md](https://github.com/handsontable/hyperformula/blob/master/CHANGELOG.md) file.
- [ ] My changes require a documentation update.
- [ ] My changes require a migration guide.
